src/priceHistory/selectors.ts

```diff
@@ -0,0 +1,16 @@
+import { createSelector } from 'reselect'
+import { RootState } from 'src/redux/reducers'
+
+export const priceHistoryPricesSelector = createSelector(
+  [(state: RootState) => state.priceHistory, (_state: RootState, tokenId: string) => tokenId],
+  (priceHistory, tokenId) => {
+    return priceHistory[tokenId]?.prices ?? []
+  }
+)
+
+export const priceHistoryStatusSelector = createSelector(
+  [(state: RootState) => state.priceHistory, (_state: RootState, tokenId: string) => tokenId],
+  (priceHistory, tokenId) => {
+    return priceHistory[tokenId]?.status ?? 'idle'
```

Comment:
seems like this should return a valid status, not 'idle', now that you've removed that status

Reply:
probly worth adding an explicit return type for this selector to avoid issues like this moving forward

Reply:
Fixed in 87377019f32ae067b8cbe445ae83677a4614236e & 9d59b1c94036c018749ced024cf12f5cfcf91213!